### ISSUE TITLE
Add penalty for over feeding pets.

### ISSUE THREAD

**Issue by egelmex** [fleure/kentpet#12]
Something like whilst pets are bloated a fat stat goes up, max health goes down or sutin.

**Comment by fleure** [comment]
Yeah that was my original idea, with some "sweet spot" of fullness that yields the best results in growth. Not sure how to implement it yet with stats/combat etc not having any implementation right now.
